src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -392,15 +392,24 @@ protected virtual void AddRecipients(TransactionBuildContext context)
         protected void AddFee(TransactionBuildContext context)
         {
             Money fee;
+            Money minTrxFee = new Money(this.network.MinTxFee, MoneyUnit.Satoshi);
```

Comment:
maybe but at the moment our feerate defaults to the fallback fee which is 10,000 per kb so the fee for a normal transaction is at 2300 satoshis. 
The problem is that stratisX doesn't relay these transactions because the minimun there is 10,000 satoshis.
We have a task to implement a fee policy, but probably not until stratisX is retired.
Try to send a trx with StratisX, the min is 10k satoshis.

Reply:
Hm, so X is so old it doesn't use it as a min fee per kb but instead it's a hardcoded min bound for tx of any size? 
Ok, then the change is good 🙂